.env

```
@@ -354,6 +354,7 @@ VITE_FEATURE_YIELDS_PAGE=true
 VITE_YIELD_XYZ_API_KEY=06903960-e442-4870-81eb-03ff3ad4c035
 VITE_FEATURE_YIELD_MULTI_ACCOUNT=true
 VITE_TON_NODE_URL=https://toncenter.com/api/v2/jsonRPC
+VITE_TEMPO_NODE_URL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_TEMPO_NODE_URL key should go before the VITE_TON_NODE_URL key

(UnorderedKey)